### PR TITLE
Mepts 1044 report name template

### PR DESCRIPTION
**Pull request by ningosi**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java [hunk 1]
```diff
@@ -1248,7 +1248,7 @@ public DataSetDefinition constructIntensiveMonitoringDataSet() {
             "startDate=${revisionEndDate-2m+1d},revisionEndDate=${revisionEndDate-1m},location=${location}"),
         "");
 
-    CohortIndicator MI15DEN16 =
+    /*CohortIndicator MI15DEN16 =
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java]: @ningosi any reason to comment this out?

Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java]: @ningosi any reason to comment this out?

**Review by psmatsinhe — COMMENTED** [review]
@ningosi any reason for commenting that code out?

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java (hunk 1 above)
Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java]: Yah, MI CAT15 will have a ticket done. that section was blocking smooth testing of the other categories. You can help by maybe uncomment and build it, then try running the report, but that will be handled in the upcoming changes for CAT 15 MI

Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/IntensiveMonitoringDataSet.java]: I will revert that back

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**